builtin.py

```diff
@@ -58,12 +59,24 @@ def advance(self, index: int) -> Node:
     def __str__(self):
         return self.title
 
+#To initialize another potion type, create another if statement in init. Define what happens on select in on_select. 
+class Potion(Node):
+    def __init__(self, title:str, desc:str, player:Player):
+        super().__init__(title, desc)
+        self.desc = desc
+        self.player = player
+        if "-health-collect" in title:
+            self.amount_health = int(title[19:21])
+            self.potionType = "Health potion"
```

Comment:
Could we have classes that inherit the potion class for Health, Speed, etc, instead of this?

Reply:
Got it. Check out the recent push. I made it as generic as possible, but notice that it is impossible to encode "this is a health potion worth 25 hp" without actually calling it out and subsequently using conditionals to decipher it. However, the implementation of Potion is certainly cleaner now.

Reply:
Could you add is_health_potion as an argument? Another approach would be to have an action method with different implementations for each class